packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md

```diff
@@ -4,13 +4,94 @@ title: Okta Identity Engine API Products release notes 2023
 
 <ApiLifecycle access="ie" />
 
+## March
+
+### Monthly release 2023.03.0
+
+| Change | Expected in Preview Orgs |
+|---------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------|
+| [Authenticator enrollment using the /authorize endpoint is GA in Production](#authenticator-enrollment-using-the-authorize-endpoint-is-ga-in-production)       |March 06, 2023   |
```

Comment:
Our convention is to use the "first time in Preview org" date here. So typically that is the "EA in Preview" date.